ConfigSpace/hyperparameters.py

```diff
@@ -689,18 +605,24 @@ def __eq__(self, other: Any) -> bool:
             return False
 
     # todo check if conversion should be done in initiation call or inside class itsel
-    def to_uniform(self, z: int = 3) -> 'UniformIntegerHyperparameter':
+    def to_uniform(self, z: np.round = 3) -> 'UniformIntegerHyperparameter':
```

Comment:
I didn't mean the type should be `np.round`, I meant you should use `np.round(int(self.mu - (z * self.sigma)))` instead of only `int(self.mu - (z * self.sigma))` to make sure that the hyperparameter is rounded correctly to the next int.